be/src/storage/segment_replicate_executor.cpp

```diff
@@ -77,6 +78,12 @@ Status ReplicateChannel::_init() {
         LOG(WARNING) << msg;
         return Status::InternalError(msg);
     }
+    _mem_tracker = ExecEnv::GetInstance()->load_mem_tracker();
```

Comment:
After this back pressure strategy, could we remove `config::brpc_socket_max_unwritten_bytes`?

Reply:
not yet, exchange node still use it.